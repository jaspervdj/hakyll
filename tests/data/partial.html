<p>This is a file that includes a partial.</p>

$partial("partial-helper.html")$
