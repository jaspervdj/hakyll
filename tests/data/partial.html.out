<p>This is a file that includes a partial.</p>

<p>This is an included partial.</p>

This is an example.


